test/TestApps/AppLifecycleTestApp/main.cpp

```diff
@@ -65,20 +65,37 @@ int main()
             else if (argument.compare(L"RegisterFile") == 0)
             {
                 ActivationRegistrationManager::RegisterForFileTypeActivation({ c_testFileExtension.c_str() },
-                    { L"open" }, L"Project Reunion Test File Type", L"logo");
+                    L"logo", L"Project Reunion Test File Type", { L"open" }, L"");
 
                 // Signal event that file was registered.
                 SignalPhase(c_testFilePhaseEventName);
                 succeeded = true;
             }
             else if (argument.compare(L"UnregisterFile") == 0)
             {
-                ActivationRegistrationManager::UnregisterForFileTypeActivation(c_testFileExtension);
+                ActivationRegistrationManager::UnregisterForFileTypeActivation({ c_testFileExtension.c_str() },
+                    L"");
 
                 // Signal event that file was unregistered.
                 SignalPhase(c_testFilePhaseEventName);
                 succeeded = true;
             }
+            else if (argument.compare(L"RegisterStartup") == 0)
+            {
+                ActivationRegistrationManager::RegisterForStartupActivation(L"this_is_a_test", L"");
+
+                // Signal event that file was registered.
+                SignalPhase(c_testStartupPhaseEventName);
+                succeeded = true;
+            }
+            else if (argument.compare(L"UnregisterStartup") == 0)
+            {
+                ActivationRegistrationManager::UnregisterForStartupActivation(L"this_is_a_test");
+
+                // Signal event that file was unregistered.
```

Comment:
>// Signal event that file was unregistered. [](start = 16, length = 43)

Is this comment significant? Seems redundant with next line

same comment applies throughout

Reply:
Since the event for each test is used multiple times for different reasons, the comment is to explain the current reason.